.github/workflows/deploy-autodokumentasjon-web.yaml

```diff
@@ -0,0 +1,40 @@
+name: "deploy: autodokumentasjon-web"
+'on':
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/deploy-autodokumentasjon-web.yaml
+      - autodokumentasjon-web/**
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy-to-cdn:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+
+      - name: Install frontend
+        working-directory: autodokumentasjon-web
+        run: npm install
+
+      - name: Build frontend
+        working-directory: autodokumentasjon-web
+        run: npm run build
+
+      - uses: navikt/frontend/actions/spa-deploy/v1@main
```

Comment:
Ser denne actionen ikke har blitt oppdatert på to år. Bør vi kanskje gå for navikt/docker-build-push slik at vi får SLSA?